LBHackney-IT/repairs-hub-frontend
# Setup google authentication

### Description of change

- Update package to include `cookie` and `jsonwebtoken`
- Setup Google authentication
- Set environment key using parameter store
- Tweak branch to test auth
- Update and fix tests to use gsso cookie
- Update readme
- Add authorisation check on pages api
- Move AuthHeader to properties
- Show logout link only when user is logged in

### Story Link

https://trello.com/c/kHrLO5Y0
